### ISSUE TITLE
Get-DbatoolsVersion to show active version

### ISSUE THREAD

**Issue by andreasjordan** [sqlcollaborative/dbatools#6739]
I often see the question "which version do you use?" and "what does `Get-Module -Name dbatools -ListAvailable` say?"

I think it would be good to have a command, that just returns the actual loaded version of dbatools.
And if the command fails, we know that the version is too old...

**Comment by wsmelton** [comment]
Asking what version is majority of cases no longer matters because we are not making massive breaking changes as we did prior to 1.0.

`Get-InstalledModule dbatools` is another command to get the newest version installed on a given machine.

If someone would like to write a Gist that has code to perform the needed check we can create a shortlink to easily share with users.

**Comment by dbaduck** [comment]
To get the version that is running should be as simple as (get-module dbatools).version. If there is something more that you want, not sure that it would take much.

**Comment by andreasjordan** [comment]
Yes, `(Get-Module dbatools).Version` is much better than `Get-Module -Name dbatools -ListAvailable`. I opened this issue after reported a not working command on slack. I told him to update dbatools and he just ran `Update-Module dbatools` and tried again - failing as well. So he missed restarting powershell or reloading the module. And even `Get-Module -Name dbatools -ListAvailable` would have reported the new version. That's why I thought that running a command would be best. Loading the latest installed version if not already loaded and showing the version.

And then I thought about myself. On my lab, I have a version of dbatools installed from PSGallery for reference and tests of bugs. But I also have a git repo to test fixes or new features. The new command could not only show the version but also, if loaded from a git repo, show the branch that was active at the time of loading. But that might be a total edge case and only relevant for me.

So summing up: A new command `Get-DbatoolsVersion` might not be as useful as I thought when opening this issue. I will think about it one or two more days and the probably closing this.
